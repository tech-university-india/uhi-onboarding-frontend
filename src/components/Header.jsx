export default function Header(props) {
    return (
        <div className="w-full bg-uhigreen-secondary">
            <div className='container mx-auto py-5 flex justify-between items-center'>
                <h1>{props.header}</h1>
            </div>
        </div>
    )
}