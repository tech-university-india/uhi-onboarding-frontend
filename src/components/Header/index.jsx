import React from 'react'

import PropTypes from 'prop-types'

export default function Header ({ text }) {
  return <header className="bg-primary shadow-header p-3 w-full rounded-md rounded-t-none">
    <h1 className="text-3xl font-light">{text}</h1>
  </header>
}

Header.propTypes = {
  text: PropTypes.string.isRequired
}
