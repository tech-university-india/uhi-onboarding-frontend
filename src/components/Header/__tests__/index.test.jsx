import React from 'react'
import Header from '..'

import { render } from '@testing-library/react'

describe('Header', () => {
  it('should render the component with text props', () => {
    const headerComponent = render(<Header />).asFragment()
    expect(headerComponent).toMatchSnapshot()
  })
})
