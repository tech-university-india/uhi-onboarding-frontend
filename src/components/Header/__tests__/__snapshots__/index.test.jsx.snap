// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render the component with text props 1`] = `
<DocumentFragment>
  <header
    class="bg-primary shadow-header p-3 w-full rounded-md rounded-t-none"
  >
    <h1
      class="text-3xl font-light"
    />
  </header>
</DocumentFragment>
`;
